src/perf/lib/SecNetPerfMain.cpp

```diff
@@ -142,10 +142,9 @@ QuicMainStart(
     QUIC_EXECUTION_CONFIG* Config = (QUIC_EXECUTION_CONFIG*)RawConfig;
     Config->PollingIdleTimeoutUs = 0; // Default to no polling.
     bool SetConfig = false;
-
-#ifndef _KERNEL_MODE
     const char* IoMode = GetValue(argc, argv, "io");
 
+#ifndef _KERNEL_MODE
```

Comment:
Actually, I _think_ the only thing that wouldn't be supported in kernel mode after all this is the `rio` stuff. Everything else can be used for kernel.

Reply:
You can exclude it for now, but depending on the final implementation, that might be common code and supported as well.